jbehave-support-core/src/test/groovy/org/jbehavesupport/test/sample/Ssh.story

```diff
@@ -21,3 +14,25 @@ Then the following data are present in [TEST] log:
 | header     | verifier    |
 | ^some.*$   | REGEX_MATCH |
 | unexpected |             |
+
+Scenario: Test log data from specific start and end timestamp
+Given log read start timestamp is set to now
+Given log read end timestamp is set to now
```

Comment:
Since its sample I would add some logic between the timestamps, or at least some comment that there is supposed to be some logic between them to make the usage more clear for users. (same for the sample below)

Reply:
Added comments to better demonstrate what it is doing